[2024-04-22T11:42:50.537+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v5.first_task manual__2024-04-22T11:42:49.463788+00:00 [queued]>
[2024-04-22T11:42:50.629+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v5.first_task manual__2024-04-22T11:42:49.463788+00:00 [queued]>
[2024-04-22T11:42:50.630+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 6
[2024-04-22T11:42:51.026+0000] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): first_task> on 2024-04-22 11:42:49.463788+00:00
[2024-04-22T11:42:51.085+0000] {standard_task_runner.py:60} INFO - Started process 528 to run task
[2024-04-22T11:42:51.118+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'first_task', 'manual__2024-04-22T11:42:49.463788+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmplbzkpxwp']
[2024-04-22T11:42:51.125+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask first_task
[2024-04-22T11:42:51.239+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-04-22T11:42:51.419+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v5.first_task manual__2024-04-22T11:42:49.463788+00:00 [running]> on host 44bc093743d3
[2024-04-22T11:42:51.936+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v5' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-04-22T11:42:49.463788+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-22T11:42:49.463788+00:00'
[2024-04-22T11:42:51.939+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-22T11:42:51.940+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2024-04-22T11:42:51.957+0000] {subprocess.py:86} INFO - Output:
[2024-04-22T11:42:51.960+0000] {subprocess.py:93} INFO - hello world, this is the first task!
[2024-04-22T11:42:51.961+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-22T11:42:52.083+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=first_task, execution_date=20240422T114249, start_date=20240422T114250, end_date=20240422T114252
[2024-04-22T11:42:52.223+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-22T11:42:52.315+0000] {taskinstance.py:3281} INFO - 2 downstream tasks scheduled from follow-on schedule check
